traffic_portal/app/src/common/service/utils/UrlUtils.js

```diff
@@ -0,0 +1,34 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+var UrlUtils = function() {
+    this.getUrlQueryParams = function(url) {
+        var vars = {};
+        var hashes = url.split('?')[1];
+        var hash = hashes != null ? hashes.split('&') : '';
```

Comment:
same as above RE: `let`/`const`

also, though, you could instead use [the `URL` object](https://developer.mozilla.org/en-US/docs/Web/API/URL), which has a `searchParams` property that implements [the `URLSearchParams` interface](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams) and provides exactly the parsing you're implementing.

Reply:
deleted this utils file and moved it all to just use the URL object
done